hi.js

```diff
@@ -902,15 +902,15 @@ settings.quality.flapAnim स्नाइपर फ्लैप एनीमे
 chall.name.hs Get {0} हैडशॉट्स {1}
 chall.name.ns Get {0} नो-स्कोप्स {1}
 chall.name.ls Get {0} लेगशॉट्स {1}
-chall.name.tmk Get {0} throwing knife kills {1}
-chall.name.fb Get {0} first bloods {1}
+chall.name.tmk Get {0} चाक़ू फेंकने से किल्स {1}
+chall.name.fb Get {0} पहला खून {1}
```

Comment:
पहला खून wording doesn't go well, maybe make it पहला किल

Reply:
okay, changing that